TLM/TLM/TLM.csproj

```diff
@@ -78,21 +78,28 @@
     <ErrorReport>prompt</ErrorReport>
     <CodeAnalysisRuleSet>..\TMPE.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
+  <PropertyGroup>
+    <SteamPath >~/Library/Application Support/Steam/</SteamPath>
+    <SteamPath Condition="! Exists ('$(SteamPath)')">$(ProgramFiles)\Steam\</SteamPath>
+    <SteamPath Condition="! Exists ('$(SteamPath)')">$(Registry:HKEY_CURRENT_USER\Software\Valve\Steam@SteamPath)</SteamPath>
+    <CSDLLPath>$(SteamPath)\steamapps\common\Cities_Skylines\Cities_Data\Managed</CSDLLPath>
```

Comment:
Suggest calling this `<ManagedDLLPath>` or something like that (regular modders will instantly know what it means as they are used to working with managed dlls)

Reply:
You probably have noticed I am not good at naming.